lib/pbench/server/database/models/api_keys.py

```diff
@@ -32,15 +35,38 @@ class APIKey(Database.Base):
     # Indirect reference to the owning User record
     user = relationship("User")
 
+    def __str__(self):
+        return f"key: {self.api_key}"
+
+    class DuplicateValue(commonDBException):
```

Comment:
The parent class, `commonDBException` requires that subclasses define `DuplicateValue()` and `NullKey()` instance methods, and the `DuplicateValue` class here does not do that.  (Ditto for the `NullKey` class at line 51.)